### PR TITLE
Added a Twitter timeline and a YouTube video embed.

### PR DESCRIPTION
**Pull request by psiwray**
The Twitter timeline can be found at the bottom of the page, right after the last section and before the footer. The YouTube embed is instead found below the preorder button.

Both the timeline and the video embed are responsive. The timeline is just centered around the screen, occupying the full width on the smaller ones. The video embed has rounded corners, so that it doesn't look too blocky. The video embed simply stretches horizontally to fill the section it's in. As the timeline, the video occupies the full width on the smaller screens.

### CODE REVIEW

**Comment by psiwray** [comment]
The browser is also complaining about a DOM attribute, `allowFullscreen`, inside the `iframe`. It suggests to change it to the same name, which is weird. For now, it isn't causing problems, but it would be nice to get rid of that error.

File: package.json [hunk 1]
```diff
@@ -16,12 +16,14 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@tailwindcss/aspect-ratio": "^0.3.0",
```
Comment (hoonsubin) [inline package.json:19]: this should be a dev dependency

File: src/templates/YouTubeEmbed.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { Section } from '../layout/Section';
+
+const YouTubeEmbed = () => (
```
Comment (hoonsubin) [inline src/templates/YouTubeEmbed.tsx:5]: make this into a component with the prop `src: string` (video source), and optional prop of `height` and `width`.
The actual prop and behavior may change depending on how it works on the page, but we want the embedded video to be a reusable component

**Review by hoonsubin — CHANGES_REQUESTED** [review]

File: src/templates/TwitterTimeline.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Timeline } from 'react-twitter-widgets';
+
+import { Section } from '../layout/Section';
+
+const TwitterTimeline = () => (
```
Comment (hoonsubin) [inline src/templates/TwitterTimeline.tsx:7]: I should have added this on my previous review, but looks like I forgot to do it.
Can you change this as a reusable component too like the YouTube component? The `screenName`, `width`, `height` should be passed as a prop.

File: src/templates/Base.tsx [hunk 4]
```diff
@@ -6,15 +6,19 @@ import { Banner } from './Banner';
...
 
 const Base = () => (
   <div className="antialiased text-primary-200">
     <Meta title={config.title} description={config.description} />
     <Navbar />
     <Hero />
+    <YouTubeEmbed src={'https://www.youtube-nocookie.com/embed/MLgFI9DVRbw'} />
     <VerticalFeatures />
     <Banner />
+    <TwitterTimeline />
```
Comment (hoonsubin) [inline src/templates/Base.tsx:21]: We should have a new section called `SocialMedia` with a header saying `Social Media` and show the Twitter timeline. We will be adding more social medial feeds later.

**Review by hoonsubin — CHANGES_REQUESTED** [review]
sorry for missing this point on my previous review

File: src/templates/TwitterTimeline.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Timeline } from 'react-twitter-widgets';
+
+type ITwitterTimelineProps = {
+  sourceType: string;
```
Comment (hoonsubin) [inline src/templates/TwitterTimeline.tsx:6]: The `sourceType` should have the type of `'url' | 'profile' | 'list'` based on the types listed on https://developer.twitter.com/en/docs/twitter-for-websites/javascript-api/guides/scripting-factory-functions

**Review by hoonsubin — CHANGES_REQUESTED** [review]

**Review by hoonsubin — APPROVED** [review]
LGTM

**Pull request merged**